packages/api/README.md

```diff
@@ -115,6 +115,10 @@ The given API has a set of three different authentication levels:
 
 The 👮 API methods are only allowed with a Magic Token, and consequently only available via https://web3.storage
 
+### Account restriction
+
+If a user's account is restricted, it means that they might have gone over the storage limit assigned to them. This restriction disables several actions such as uploading files, adding and replacing pin requests, or publishing a name record. Note that even if the account has [pinning service API access](https://docs.web3.storage/how-tos/pinning-services-api/#requesting-access), account restriction will disable adding and replacing of pins. It is however still possible to delete pins and create/delete API tokens. For more information, please email <support@web3.storage>.
```

Comment:
> It is however still possible to delete pins and create/delete API tokens.

I'm new to this piece of work, so probably missing some context, but I'm wondering why we would still allow folks to delete pins and create/delete api tokens when their account is locked?

Reply:
@olizilla This is to allow them to delete files to go back below their quota. Also, if their token was compromised and abused, they can create a new token and block that token and clean up their account.